pycti/api/opencti_api_client.py

```diff
@@ -94,7 +117,18 @@ def __init__(self, url, token, log_level="info", ssl_verify=False, proxies={}):
         numeric_level = getattr(logging, self.log_level.upper(), None)
         if not isinstance(numeric_level, int):
             raise ValueError("Invalid log level: " + self.log_level)
-        logging.basicConfig(level=numeric_level)
+
+        logging.error(f"TESTS : {json_logging=}")
```

Comment:
Can you please remove this line. I think that's a left over from your testing phase

Reply:
Sorry about that, removed.